README.md

```diff
@@ -1,11 +1,10 @@
-# MembraneRpicamPlugin
+# Membrane Rpicam Plugin
 
-**TODO: Add description**
+Membrane Rpicam Plugin allows capturing video from RaspberryPi camera using [rpicam-apps](https://github.com/raspberry-pi/rpicam-apps).
 
 ## Installation
 
-If [available in Hex](https://hex.pm/docs/publish), the package can be installed
-by adding `membrane_rpicam_plugin` to your list of dependencies in `mix.exs`:
+The package can be installed by adding `membrane_rpicam_plugin` to your list of dependencies in `mix.exs`:
```

Comment:
Isn't there something more to install on Raspberry for this to work, compared to a pure elixir app?